### PR TITLE
Making roughness ranges all [0,1]

### PR DESCRIPTION
**Pull request by portsmouth**
As discussed in https://github.com/AcademySoftwareFoundation/OpenPBR/issues/143, this PR fixes up the stated roughness ranges to forbid roughness > 1.  It isn't useful to allow the roughness to exceed a soft-max at 1, as the appearance is unconvincing  (looks like fabric, as the microfacets are mostly vertical -- which is not "rough", and we have a dedicated fuzz layer for this kind of appearance).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -713,7 +713,7 @@
 \begin{equation}
 V_d = \frac{\mathtt{transmission\_dispersion\_abbe\_number}} {\mathtt{transmission\_dispersion\_scale}} \ .
 \end{equation}
-At the default **`transmission_dispersion_scale`** of zero, the Abbe number is infinite, which corresponds to no dispersion. At the maximum of 1, the Abbe number peaks at $V_d$ = **`transmission_dispersion_abbe_number`** which defaults to 20.
+At the default **`transmission_dispersion_scale`** of zero, the Abbe number is infinite, which corresponds to no dispersion. At the maximum of 1, the Abbe number peaks at $V_d = \mathtt{transmission\_dispersion\_abbe\_number}$ which defaults to 20.
```
Comment (virtualzavie) [inline index.html:716]: Looks like this change is conflicting between #157 and here.

**Review by virtualzavie — APPROVED** [review]
LGTM.
Approved, pending resolution of the conflict between #157 and this one.

**Comment by portsmouth** [comment]
> LGTM. Approved, pending resolution of the conflict between #157 and this one.

Done in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/160/commits/6a164f73df27e4473739cafa4852f77e32b525df.

**Review by AdrienHerubel — APPROVED** [review]
LGTM

**Pull request merged**